scripts/rnnlm/prepare_rnnlm_dir.sh

```diff
@@ -58,7 +58,7 @@ if [ $stage -le 1 ]; then
     done
   fi
 
-  rnnlm/get_special_symbol_opts.py < $dir/config/words.txt > $dir/special_symbol_opts.txt
+  cat $dir/config/words.txt | grep '<' | rnnlm/get_special_symbol_opts.py > $dir/special_symbol_opts.txt
```

Comment:
Hainan, is this change supposed to be here?  I thought we fixed this problem in a different way.

Reply:
Oh I was not aware this was fixed. I guess I will change this.

Reply:
check that it still works though.